Rakefile

```diff
@@ -4,3 +4,45 @@
 require File.expand_path('../config/application', __FILE__)
 
 Rails.application.load_tasks
+
+namespace :unicorn do
+  desc "Start unicorn for development env."
+  task(:start) {
+    config = rails_root + "config/unicorn.rb"
+    sh "bundle exec unicorn_rails -c #{config} -E production -D -p 8080"
+  }
+
+  desc "Stop unicorn"
+  task(:stop) { unicorn_signal :QUIT }
+
+  desc "Restart unicorn with USR2"
+  task(:restart) { unicorn_signal :USR2 }
+
+  desc "Increment number of worker processes"
+  task(:increment) { unicorn_signal :TTIN }
+
+  desc "Decrement number of worker processes"
+  task(:decrement) { unicorn_signal :TTOU }
+
+  desc "Unicorn pstree (depends on pstree command)"
+  task(:pstree) do
+    sh "pstree '#{unicorn_pid}'"
+  end
+
+  def unicorn_signal signal
+    Process.kill signal, unicorn_pid
+  end
+
+  def unicorn_pid
+    begin
+      File.read("/tmp/unicorn.pid").to_i
+    rescue Errno::ENOENT
+      raise "Unicorn doesn't seem to be running"
+    end
+  end
+
+  def rails_root
+    require "pathname"
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)